stdlib/src/collections/deque.mojo

```diff
@@ -274,6 +506,37 @@ struct Deque[ElementType: CollectionElement](
         if self._head == self._tail:
             self._realloc(self._capacity << 1)
 
+    fn appendleft(inout self, owned value: ElementType):
```

Comment:
Why it is called `appendleft` and not `appendleft`? Mojo style guide points that snake_case should be used :)

Reply:
This is how it named in Python: https://docs.python.org/3/library/collections.html#collections.deque
Do we want to follow Python API, or should we follow the Mojo guides?
Rust and C++ call it `push_front` and `push_back`, so lots of choices if we do not want to follow Python :)

Reply:
Let's follow the Python APIs for the time being and we can discuss this in our next team design meeting (CC: @ConnorGray).